docs/robots/husky/maintenance_husky.mdx

```diff
@@ -23,6 +74,90 @@ import ComponentUpdatingPumaFirmware from "/components/maintenance/firmware_upda
 
 <ComponentUpdatingPumaFirmware />
 
+## Lead Acid Battery Care
+
+Husky's default power supply is a sealed 24 V lead acid battery pack (VRLA), providing 20 Ah of charge.
+These tips are intended to help keep your Husky Lead-Acid Battery in tip-top shape.
+With proper maintenance, the battery should maintain the majority of its capacity for hundreds of cycles.
+
+The most damaging thing to a lead-acid battery is a phenomenon called sulfation.
+When a lead-acid battery is left in an uncharged state for long periods of time, sulfate 
+crystals solidify on the electrodes of the battery.
+This effect can be permanent, and it causes premature reduction in capacity.
+Therefore, it is important to fully charge the battery as soon as you are finished using it,
+regardless of how much capacity remains.
+
+Your Husky ships with a three-stage battery charger that completely charges a Husky battery.
+Charging occurs in three stages: Constant current, topping charge, and float charge.
+The bulk of the capacity (70%) is regained during the first stage.
+
+It is important to let the charger complete all three stages whenever possible, to help maintain the life of the battery.
+The initial stage will take 5-7 hours, and a complete charge may take up to 14 hours.
+
+<center>
+  <img
+    src="/img/robot_images/husky_images/robot_husky_18.png"
+    width="600"
+  />
+</center>
+
+In general, following these tips will help maximize the life of the battery:
+
+- Always fully charge the battery as soon as you are finished using it.
+- Charge batteries at room temperature. Never charge lead-acid batteries at temperatures above 35 degrees Celcius.
+- Charge batteries in a well-ventilated area.
+- Do not allow the battery to freeze. A low battery will freeze sooner than a fully charged one. Never charge a frozen battery.
+- Stored batteries should be topped up every month. Completely cycle batteries every 4-6 months.
+- At a discharge rate of 50% DOD (depth of discharge), the lifecycle of the Husky battery is 450-550 cycles.
+  After this point, reduced capacity will be apparent.
+- Regularly discharging the battery below 50% DOD will reduce the lifecycle of the battery, sometimes to less than 300 cycles.
+
+## Lithium Ion Battery Care
+
+If your Husky has an upgrade to Lithium Ion batteries, note the required maintenance below.
```

Comment:
Is the capitalized Lithium Ion intentional? I'm indifferent, but you could consider italics or bold font to draw emphasis.